front/components/AmountChooser.js

```diff
@@ -57,6 +57,31 @@ export default class AmountChooser extends React.Component {
 
         return (
             <div className="amount-chooser">
+
+                <div className="amount-chooser__monthly form__radio">
+                    <input
+                        type="radio"
+                        name="abonnement"
+                        id="donation-one-time"
+                        value="0"
+                        defaultChecked={!this.props.abonnement} />
+
+                    <label htmlFor="donation-one-time" id="donation-one-time_label">
```

Comment:
do we really need to have `id="..."` on label?

Reply:
not necessary, i removed them